source/common/event/scaled_range_timer.h

```diff
@@ -70,7 +70,7 @@ class ScaledRangeTimerManager {
     float value_;
   };
 
-  static constexpr int kBucketScaleFactor = 2;
+  static constexpr float kBucketScaleFactor = 1.41421356237; // = sqrt(2)
```

Comment:
You may want to use log base 2, and then multiply the log result by some number.  Multiplying by 2 would be equivalent to using sqrt(2) as the base, multiplying by 0.5 would be equivalent to using a log base of 4.